src/App.js

```diff
@@ -38,6 +39,10 @@ function App() {
           <Route path="/signup" element={<Signup />} />
           <Route path="/look" element={<Look />} />
           <Route path="/record" element={<WithLogin children={<Record />} />} />
+          <Route
+            path="/updataPassword"
```

Comment:
```suggestion
            path="/user/update/password"
```

url에는 대문자 안 쓰는 게 원칙입니당